custom/covid/management/commands/run_all_management_command.py

```diff
@@ -0,0 +1,48 @@
+import csv
+from gevent.pool import Pool
+
+from django.core.management.base import BaseCommand
+from django.core.management import call_command
+
+from corehq.apps.linked_domain.dbaccessors import get_linked_domains
+
+
+DEVICE_ID = __name__ + ".run_all_management_commands"
+
+
+class Command(BaseCommand):
+
+    def add_arguments(self, parser):
+        parser.add_argument('domain')
+        parser.add_argument('location-csv')
+        parser.add_argument('--and-linked', action='store_true', default=False)
+
+    def handle(self, domain, location_csv, **options):
+        domains = {domain}
```

Comment:
I think it makes more sense to get this info from the csv (and just handle anything you pass in the csv) rather than fetching domains here and comparing them to what is in the file.

Reply:
With latest commit, it now outputs something like `SUCCESS: update_owner_ids command for cases-domain` or `COMMAND FAILED: ['Invalid username: None'] exception while running add_hq_user_id_to_case for cases-domain` after each command is run.